File: 4th Semester/Design and programming environments/Project Java Client-Server/Networking/src/main/java/dto/InscriereDTO.java
package dto;

import domain.Participant;
import domain.Proba;

import java.io.Serializable;

public class InscriereDTO implements Serializable {
    private Participant p;
    private Proba[] probas;

    public InscriereDTO(Participant p, Proba[] probas) {
        this.p = p;
        this.probas = probas;
    }

    public Participant getP() {
        return p;
    }

    public void setP(Participant p) {
        this.p = p;
    }

    public Proba[] getProbas() {
        return probas;
    }

    public void setProbas(Proba[] probas) {
        this.probas = probas;
    }
}

File: 4th Semester/Design and programming environments/Proiect C-Sharp Client-Server Sockets/AppNetworking/DTO/InscriereDTO.cs
using AppModel;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppNetworking.DTO
{
    [Serializable]
    public class InscriereDTO
    {
        public InscriereDTO(Participant a, Proba b)
        {
            Participant = a;
            Proba = b;
        }
        public Participant Participant { get; set; }

        public Proba Proba { get; set; }
        public int id { get; set; }
    }
}
